tests/utils.py

```diff
@@ -320,7 +320,11 @@ def get_proxy(self):
         proxy.start()
         return proxy
 
-    def generate_block(self, numblocks=1):
+    # wait_for_mempool can be True or a positive integer to wait for the
+    # amount of transactions in the mempool before generating blocks.
+    def generate_block(self, numblocks=1, wait_for_mempool=0):
```

Comment:
We might want to go a step further and not just check for the number of transactions, but also the specific transaction IDs we'd like to include. So this could become 
```suggestion
    def generate_block(self, numblocks=1, must_include=None):
```

And we call it with a list of transaction IDs that we check for.

Reply:
okay, will do :D